Jenkinsfile.sdk

```diff
@@ -210,12 +210,12 @@ pipeline {
 						}
 					}
 				}
-				stage('Lisk Framework Plugins') {
+				stage('Lisk Framework Plugins Unit') {
 					agent { node { label 'lisk-framework' }}
 					steps {
 						setup()
 						nvm(getNodejsVersion()) {
-							sh 'npx lerna run test:ci --scope=@liskhq/lisk-framework-*'
+							sh 'npx lerna run test:unit --scope=@liskhq/lisk-framework-*'
```

Comment:
I think jenkins wise, it uses different resource, so it's better to keep it generic.
If it fails, you should be able to see easily in the log because it says the package name. (similar to elements)